Apps/WebClient/src/Server/Controllers/BetaRequestController.cs

```diff
@@ -85,7 +87,7 @@ public IActionResult CreateBetaRequest([FromBody] BetaRequest betaRequest)
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpGet]
         [Route("{hdid}")]
-        [Authorize(Policy = UserPolicy.Read)]
+        [Authorize(Policy = UserProfilePolicy.Write)]
```

Comment:
@FuriousLlama can you confirm if this should be Read instead of Write as it is for "GetBetaRequest" right?

Reply:
read sounds more appropiate

Reply:
Updated